tests/test_pdf_qgauss.py

```diff
@@ -0,0 +1,60 @@
+#  Copyright (c) 2024 zfit
+import numpy as np
+import pytest
+import zfit
+from zfit import z
+import zfit.z.numpy as znp
+from numba_stats import qgaussian as qgaussian_numba
+import tensorflow as tf
+
+mu_true = 0.5
+sigma_true = 1.1
+true_integral_dict = {1.00001: 0.9999782, 1.5: 0.9695, 2.0: 0.8063152, 2.5: 0.4805167, 2.9: 0.1078687, 2.99: 0.011044}
```

Comment:
How was this obtained? Maybe a comment, in case things change? I assume numerically?